distro/pkg/rpm/libnetconf2.spec

```diff
@@ -28,27 +28,22 @@ servers. NETCONF is the NETwork CONFiguration protocol introduced by IETF.
 
 %prep
 %autosetup -p1
-mkdir build
 
 %build
-cd build
-cmake \
-    -DCMAKE_INSTALL_PREFIX:PATH=%{_prefix} \
-    -DCMAKE_BUILD_TYPE:String="Release" \
-    -DCMAKE_C_FLAGS="${RPM_OPT_FLAGS}" \
-    -DCMAKE_CXX_FLAGS="${RPM_OPT_FLAGS}" \
-    ..
-make
+%cmake
```

Comment:
`-DCMAKE_BUILD_TYPE=RELWITHDEBINFO` is not needed, does the macro supply any specific build type? Maybe just for consistency it should be added anyway, was required for the *sysrepo* package.